examples/llm/icl_eval/yamls/mosaic_gpt_eval.yaml

```diff
@@ -13,18 +13,15 @@ model:
   n_layers: 12
   mlp_ratio: 4
   max_seq_len: 2048
```

Comment:
do `${max_seq_len}` instead?
(gotta set it above tho)

also `${tokenizer_name}`.

Reply:
Oh this is actually a bigger issue b/c right now these values reference the `v0.0.3` branch. At the end of all `v0.0.4` changes we will have to self-reference the tag. I will update the stuff you mentioned now but this is expected to be broken on the release branch right now.